### PR TITLE
Add personal security guide

### PR DESCRIPTION
**Pull request by charlietran**
Some notes on the basics of how personal digital security should be handled, again inheriting from thoughtbot/guides

### CODE REVIEW

File: security/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Failing to follow these guidelines will likely put you, your team, and your
+deployed services at risk of compromise or loss of privacy.
+
+Using Passwords
+---------------
+* Use a unique password for every account you create.
+* Use a tool such as [1password](https://agilebits.com/onepassword) to generate random passwords.
+* Avoid sharing passwords in plaintext. If you must share a password in Slack,
+  delete the message immediately after you confirm it has been received.
```
Comment (ksheurs) [inline security/README.md:16]: Maybe include a reference to Keybase.io here too?

Comment (charlietran) [reply security/README.md:16]: I'm not totally sold on Keybase after talking with George from Thoughtbot a bit about it, but we could definitely level up our PGP usage in general. For now I think I'd actually prefer sharing via 1password if necessary

Comment (ksheurs) [reply security/README.md:16]: Gotch ya. What are some of the points against Keybase? Still new to it myself.

**Comment by jamiew** [comment]
Isn't Keybase for public keys, so basically just useful for email? And maybe OTR chats

**Comment by ksheurs** [comment]
@jamiew you can also do private/shared folders.

**Comment by charlietran** [comment]
Keybase is basically hosted PGP. The issue was that you basically
reduce the inherent security of PGP by hosting your private keys on the
service, and Keybase doesn't add *that much* on top of it.

**Pull request merged**